pkg/storaged/part-tab.jsx

```diff
@@ -18,29 +18,54 @@
  */
 
 import React from "react";
+import { Alert } from "@patternfly/react-core/dist/esm/components/Alert/index.js";
 import { DescriptionList, DescriptionListDescription, DescriptionListGroup, DescriptionListTerm } from "@patternfly/react-core/dist/esm/components/DescriptionList/index.js";
+import { StorageButton } from "./storage-controls.jsx";
+import { get_resize_info, free_space_after_part, grow_dialog, shrink_dialog } from "./resize.jsx";
 
 import cockpit from "cockpit";
 import * as utils from "./utils.js";
 
 const _ = cockpit.gettext;
 
-export class PartitionTab extends React.Component {
-    render() {
-        const block_part = this.props.client.blocks_part[this.props.block.path];
+export const PartitionTab = ({ client, block, warnings }) => {
+    const block_part = client.blocks_part[block.path];
+    const unused_space_warning = warnings.find(w => w.warning == "unused-space");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19385-20230925-111150-44391ef2-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)